Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/DirectorySymlinkTests.cs

```diff
@@ -94,9 +96,19 @@ protected override void Dispose(bool disposing)
 Versions/
 Versions/A/
 Versions/A/file
-Versions/A/sym-loop -> ../sym-A
-Versions/sym-A -> A
-Versions/sym-sym-A -> sym-A
+Versions/A/sym-loop -> ../sym-A/
+Versions/sym-A -> A/
+Versions/sym-sym-A -> sym-A/
+";
+
+        private const string DirectoryLayoutWinTest = @"

```

Comment:
how's this layout different from the one above?

Reply:
since the window native can not deal with symlink loop